zetaclient/config/types.go

```diff
@@ -148,6 +151,30 @@ type Config struct {
 	mu *sync.RWMutex
 }
 
+// Validate performs basic configuration validation.
```

Comment:
Context on these changes? we have validation logic added here https://github.com/zeta-chain/node/pull/4361 we might want to avoid conflicts